src/common/transformers/LOWERCASE.js

```diff
@@ -0,0 +1,14 @@
+import { transformer } from './transformer';
+
+export const valueToLowerCase = value =>
+    typeof value === 'string' ? value.toLowerCase() : value;
+
+const transformation = () => value => transformer(valueToLowerCase, value);
+
+transformation.getMetas = () => ({
+    name: 'LOWERCASE',
+    type: 'transform',
+    args: [],
+});
+
+export default transformation;
```

Comment:
is it very useful ?

Reply:
Maybe not *very* useful in this case, but... if every `.js` file could have a `.spec.js` counterpart, it would be easy to know that everything is tested.

Reply:
I added it ;-)